### PR TITLE
Swapping correct sweepers axes with singleshot averaging_mode

### LINKED ISSUE #701: `np.moveaxis` changing wrong indexes for singleshot on Zurich driver
For singleshot results, indexes in `rearrange_sweepers` function are incorrect.

### PR DESCRIPTION
**Pull request by GabrielePalazzo**
Closes #701.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by Jacfomg — APPROVED** [review]
Looks good to me, thanks @GabrielePalazzo

File: src/qibolab/instruments/zhinst.py [hunk 1]
```diff
@@ -1055,24 +1057,30 @@ def fast_reset(self, exp, qubits, fast_reset):
...
+            sweepers (list[Sweeper]): list of sweepers used in the experiment.
+
+        Returns:
+            rearranging_axes (np.ndarray): array of shape (2,) and dtype=int containing
+                the indexes of the sweepers to be swapped. Defaults to np.array([0, 0])
+                if no swap is needed.
+            sweepers (list[Sweeper]): updated list of sweepers used in the experiment. If
+                sweepers must be swapped, the list is updated accordingly.
+        """
+        rearranging_axes = np.zeros(2, dtype=int)
```
Comment (stavros11) [inline src/qibolab/instruments/zhinst.py:1076]: I am assuming that this driver supports up to two sweepers only, right?

**Review by stavros11 — APPROVED** [review]
Result shape tests are now passing with Zurich, thanks @GabrielePalazzo.

File: src/qibolab/instruments/zhinst.py (hunk 1 above)
Comment (GabrielePalazzo) [reply src/qibolab/instruments/zhinst.py:1076]: No, more sweepers are supported, but in practice that function was already considering using only two sweepers.

**Pull request merged**